addons/mail/static/src/components/message/message.xml

```diff
@@ -173,12 +191,12 @@
                             <ul class="o_Message_trackingValues">
                                 <t t-foreach="trackingValues" t-as="value" t-key="value.id">
                                     <li>
-                                        <div class="o_Message_trackingValue">
+                                        <div class="o_Message_trackingValue" aria-atomic="true">
                                             <div class="o_Message_trackingValueFieldName o_Message_trackingValueItem" t-esc="value.changed_field"/>
                                             <t t-if="value.old_value">
                                                 <div class="o_Message_trackingValueOldValue o_Message_trackingValueItem" t-esc="value.old_value"/>
                                             </t>
-                                            <div class="o_Message_trackingValueSeparator o_Message_trackingValueItem fa fa-long-arrow-right" title="Changed" role="img"/>
+                                            <div class="o_Message_trackingValueSeparator o_Message_trackingValueItem fa fa-long-arrow-right" title="Changed" role="img" tabindex="-1"/>
```

Comment:
What's the point of tabindex on the separator? Especially if it doesn't hold any other value, such as aria-label?
Maybe this should have sr only text, such as "from...to" instead?

Reply:
The img role is making this element focusable, which is useless as it has no associated action.

Reply:
Ok, but there is no way everybody will know that and not break it in the future.

Reply:
Maybe the `role="img"` should not have been on this element in the first place; I'd say that the only the group (old value, separator, new value) as a whole is meaningful.

Assuming I understand how `role="img"` works, I think we want something like that:
```xml
<div role="img" t-attf-aria-label="Changed from {{ value.old_value }} to {{ value.new_value }}">
    <div class="o_Message_trackingValueOldValue o_Message_trackingValueItem" t-esc="value.old_value"/>
    <div class="o_Message_trackingValueSeparator o_Message_trackingValueItem fa fa-long-arrow-right"/>
    <div class="o_Message_trackingValueNewValue o_Message_trackingValueItem" t-esc="value.new_value"/>
</div>
```